WMFComponents/Sources/WMFComponents/Components/Activity Tab/WMFActivityTabViewModel.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+import SwiftUI
+import WMFData
+
+@objc public class WMFActivityTabViewModel: NSObject, ObservableObject {
+    var localizedStrings: LocalizedStrings
+    var dataController: WMFActivityTabDataController
```

Comment:
Let's start this with `private let`, we can expose it more later if you find you need it mutable / accessible outside the class.